12.0.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.5-stretch AS base
+FROM python:3.7-stretch AS base
```

Comment:
:-1: to this change. We have discussed already about this.

Reply:
Yes, I thought about [what you said](https://github.com/Tecnativa/doodba/pull/168#discussion_r219116920):

> we will be able to develop in our stack things that are incompatible with the rest of Odoo because we don't remember that are 3.6 or 3.7 features

I think you didn't realize the opposite problem is also true: other people can do exactly the same, and we'd be affected because we are the ones using an older Python version.

If I have to choose among being affected by a problem or affecting others, I think the sanest approach, sadly, is affecting others.